schema.sql

```diff
@@ -5,4 +5,4 @@ CREATE TABLE animals (
     escape_attemps integer,

```

Comment:
Inside the schema file, you should add a column named species to the animals table as mentioned in the project requirements 👍😀
kindly fix this issue👍.